avm/res/app-configuration/configuration-store/version.json

```
@@ -1,7 +1,7 @@
 {
     "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
-    "version": "0.1",
+    "version": "0.1.1",
```

Hey @JFolberth, no need to update the patch version, since that in Bicep is done automatically by the CI.

However, as per semver, given the addition of a child resource, and with that a new feature, I'd suggest bumping the minor version
```suggestion
    "version": "0.2",
```